### ISSUE TITLE
How to resume tracking annotation

### ISSUE THREAD

**Issue by castrovictor** [openvinotoolkit/cvat#3559]
I have trying to figure something out but I am stuck. Let say we start draw the bounding box for an object along consecutive frames, using interpolation. We have done that from frame 100 to 200, and clicked the save button. If later on I want to continue labeling the object, from frames 201 to 300, I can do it as before, but the id of the label would be different. How do I continue working keeping the same id? I do not see any "resume" or "edit" button or any workaround. Thanks in advance.

**Comment by bsekachev** [comment]
Hi,
Do you mean that objects IDs can be changed after reopening the job?

**Comment by castrovictor** [comment]
> Hi,
> Do you mean that objects IDs can be changed after reopening the job?

Yes. If the video is too long I would like to save and take breaks instead of tracking the object from the beginning to the end

**Comment by bsekachev** [comment]
- These IDs, you see, are assigned automatically to each object when you open a job (just incrementally). 
- Their main purpose is to help to implement client-side business logic in JavaScript code during one session.
- Each object saved on the server also has a server ID (assigned by a database). Server id is not changed between sessions (but a user almost does not see it, only when creates a link to an object).

I agree that client ID does not bring a lot of benefits to a user, because of its changes. But I also hardly imagine what benefit could bring server ID (if we display it instead of clientID) to a user when annotation. 

To sum up, when you reload a job, client IDs can be changed, but actually, you continue working with the same object in the case you described.

**Comment by castrovictor** [comment]
Perfect, it makes sense then. Thank you!!
